src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,364 @@
+package duke;
+
+import duke.task.Task;
+
+/**
+ * Stores commands, descriptors, responses and error messages used by duke.Duke chatbot.
+ */
+public class Ui {
+    /** Enums for Duke chatbot descriptors */
+    public enum Descriptors {
+        AT("at"),
+        BY("by");
+
+        private final String descriptor;
+
+        Descriptors(String descriptor) {
+            this.descriptor = descriptor;
+        }
+
+        public String getDescriptor() {
+            return this.descriptor;
+        }
+
+        public int getLength() {
+            return this.descriptor.length();
+        }
+
+        @Override
+        public String toString() {
+            return this.descriptor;
+        }
+    }
+
+    /** Enums for Duke chatbot commands */
+    public enum Commands {
+        TODO("todo"),
+        DEADLINE("deadline"),
+        EVENT("event"),
+        DONE("done"),
+        DELETE("delete"),
+        LIST("list"),
+        DATE("date"),
+        FIND("find"),
+        BYE("bye");
+
+        private final String command;
+
+        Commands(String command) {
+            this.command = command;
+        }
+
+        public String getCommand() {
+            return this.command;
+        }
+
+        public int getLength() {
+            return this.command.length();
+        }
+
+        @Override
+        public String toString() {
+            return this.command;
+        }
+    }
+
+    /**
+     * Returns the standard welcome response.
+     *
+     * @return Standard welcome response.
+     */
+    public String getWelcome() {
+        return "Hello! I'm Duke\nWhat can I do for you?";
+    }
+
+    /**
+     * Returns the standard goodbye response.
+     *
+     * @return Standard goodbye response.
+     */
+    public String getGoodbye() {
```

Comment:
Maybe for this method (and similar methods in this class), you could rename the methods to this format: 
"<command>Message". So for this case, maybe something like "goodbyeMessage". Feel free to use or not use my suggesstion though!

Reply:
Thanks for the suggestion! I agree that this will provide a better description for the function's purpose and I will make relevant changes in the next push.